src/containers/edit-tutor-profile/NotificationContainer.tsx

```diff
@@ -0,0 +1,53 @@
+import Box from '@mui/system/Box'
+import Typography from '@mui/material/Typography'
+import Switch from '@mui/material/Switch'
+
+import { useTranslation } from 'react-i18next'
+
+import SettingItem from '~/components/setting-item/SettingItem'
+import { styles } from '~/containers/edit-tutor-profile/NotificationContainer.style'
+export const NotificationContainer = () => {
+  const { t } = useTranslation()
+
+  return (
+    <Box sx={styles.root}>
+      <Typography sx={styles.title}>
```

Comment:
can you use titleWithDescription component here?

Reply:
yeah, that's better thanks